lib/cli.js

```diff
@@ -279,6 +279,17 @@ async function translateOptions(
 		options.warnIgnored = warnIgnored;
 		options.flags = flag;
 
+		/*
+		 * If there are any flags set in the environment, prepend them to the list of flags
+		 * passed in via the command line. This allows users to set flags in their environment
+		 * and have them apply to all ESLint runs.
+		 */
+		if (process.env.ESLINT_FLAGS) {
+			const envFlags = process.env.ESLINT_FLAGS.split(/\s*,\s*/gu);
+			const cliFlags = options.flags || [];
+			options.flags = Array.from(new Set([...envFlags, ...cliFlags]));
+		}
+
```

Comment:
Since the environment variable is used only in the ESLint CLI, does this solve the use case from https://github.com/eslint/eslint/issues/19100? The original post of https://github.com/eslint/eslint/issues/19100 mentions IDEs, which typically don't use ESLint CLI.

Reply:
Ah, I missed that. I guess it would need to be in `ESLint` then.